packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/upgrade/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,9 @@
+{
+  "dedicatedCloud_datacenter_backup_upgrade_title": "Changement de l’offre de sauvegarde",
+  "dedicatedCloud_datacenter_backup_upgrade_confirm": "Passer à l’offre {{ name }}",
+  "dedicatedCloud_datacenter_backup_upgrade_cancel": "Annuler",
+  "dedicatedCloud_datacenter_backup_upgrade_description": "Vous souhaitez changer votre offre de sauvegarde de l’offre <strong>{{ name }}</strong>.",
```

Comment:
You can remove `strong` tag in the translation, and use it when you pass the value in `translate-values` directive

Reply:
I dont think we can pass the `html` part in `translate-values`. This looks cleaner anyway :)